### PR TITLE
✨Code-Cleanups and Enhancing README

### PR DESCRIPTION
**Pull request by Lebyy**

### CODE REVIEW

**Comment by EhsanFox** [comment]
Readme and stuff are unnecessary!

**Comment by Lebyy** [comment]
> Readme and stuff are unnecessary!

1. The current code errors had alot of grammar errors.
2. The README wasn't convincing enough
3. This PR makes the code easy to read.

**Comment by Lebyy** [comment]
oops didn't mean to close it

**Review by AngeloCore — CHANGES_REQUESTED** [review]
Check

File: src/index.js [hunk 1]
```diff
@@ -1,49 +1,50 @@
...
-module.exports.MessageActionRow = require('./v12/Classes/MessageActionRow');
-module.exports.ButtonInteraction = require('./v12/Classes/INTERACTION_CREATE');
\ No newline at end of file
+const { Structures, Client } = require("discord.js");
+
+var version = require("discord.js").version.split("");
+version = parseInt(version[0] + version[1]);
+
+module.exports = (client) => {
+  if (version < 11 || version === 11) {
```
Comment (honkling) [inline src/index.js:7]: Why not do version <= 11 instead?

**Comment by AngeloCore** [comment]
@Lebyy so?

**Comment by AngeloCore** [comment]
Inactive

**Pull request closed without merging**